File: Week26/day4/dch/dch.py
# Instructions
# Write a function to compute 5/0 and use try/except to catch the exceptions.
# Bonus : use some Buit-in exceptions of Python : Look here



def compute():
    try: 
        num1 = int(input('Enter first number: '))
        num2 = int(input('Enter second number: '))
        res = num1/num2

        print(res)
    except ZeroDivisionError:
       print("you cant divide by zero")
    except TypeError:
       print("typeerror - use numbers")
    except ValueError:
       print("valueerror - invalid entry")


compute()

File: Week17/day5/TS/src/dch/dch.ts
function processInput(input: number | string | boolean): number | string | boolean {
    if (typeof input === 'number') {
      return input * input; 
    } else if (typeof input === 'string') {
      return input.toUpperCase(); 
    } else if (typeof input === 'boolean') {
      return !input; 
    } else {
      return 'Invalid input type'; 
    }
}

console.log(processInput(5));        
console.log(processInput("hello"));  
console.log(processInput(true)); 